modules/qt/hm.nix

```
@@ -64,7 +64,7 @@
 
       home.packages = lib.optional (config.qt.style.name == "kvantum") kvantumPackage;
 
-      qt = {
+      qt = lib.mkDefault {
```

Due to https://github.com/danth/stylix/pull/388, we should not use `lib.mkDefault` to make the workaround slightly more convenient:

```suggestion
      qt = {
```